hazelcast-jet-core/src/main/java/com/hazelcast/jet/Sources.java

```diff
@@ -137,9 +137,10 @@ private Sources() {
      * singleAttribute()} and {@link
      * com.hazelcast.projection.Projections#multiAttribute(String...)
      * multiAttribute()}), depending on how the data is stored in the map.
-     * Similarly, using the {@link com.hazelcast.query.PredicateBuilder
-     * PredicateBuilder} will build a predicate that makes use of any indexes
-     * created on the map.
+     * Similarly, {@link com.hazelcast.query.PredicateBuilder
+     * PredicateBuilder} or {@link com.hazelcast.query.Predicates} can be used
+     * to only apply the predicate to a single attribute without deserializing the
+     * whole object. Use of map indices is not currently supported.
```

Comment:
should we add a comment about `InMemoryFormat.Portable` ? User will be surprised if there is still de-serialization going on even if s/he used `om.hazelcast.query.Predicates`

Reply:
You mean `InMemoryFormat.BINARY` ?

Reply:
Yes, to avoid deserializing one should use `Portable` objects if map configured with `InMemoryFormat.BINARY` or `InMemoryFormat.NATIVE`

Reply:
I updated the section before to add a link to `InMemoryFormat`.